src/encord_active/lib/common/image_utils.py

```diff
@@ -43,11 +43,13 @@ def draw_object(
     hex_color = color.value if isinstance(color, Color) else color
     _color: Tuple[int, ...] = hex_to_rgb(hex_color)
     _color_outline: Tuple[int, ...] = hex_to_rgb(hex_color, lighten=-0.5)
-    if isinstance(row["rle"], str):
-        if with_box:
-            box = get_bbox_csv(row)
-            image = cv2.polylines(image, [box], isClosed, _color, thickness // 2, lineType=cv2.LINE_8)
 
+    has_mask = isinstance(row["rle"], str)
+    if with_box or not has_mask:
+        box = get_bbox_csv(row)
+        image = cv2.polylines(image, [box], isClosed, _color, thickness, lineType=cv2.LINE_8)
+
+    if has_mask:
```

Comment:
[nitpick] 💅 

An early return makes it read slightly better

```suggestion
    if with_box or not has_mask:
        box = get_bbox_csv(row)
        image = cv2.polylines(image, [box], isClosed, _color, thickness, lineType=cv2.LINE_8)
        return image

```

Reply:
@Encord-davids fixed this